samples/filterSnippets.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// sample-metadata:
+//   title: Read a row
+//   description: Get all the data for a Bigtable row by key.
+//   usage: node readRow.js <instanceId>
+
+function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  filterType = 'filterRowSample'
+) {
+  // [START bigtable_filters_limit_row_sample]
+  // [START bigtable_filters_limit_row_regex]
+  // [START bigtable_filters_limit_cells_per_col]
+  // [START bigtable_filters_limit_cells_per_row]
+  // [START bigtable_filters_limit_cells_per_row_offset]
+  // [START bigtable_filters_limit_col_family_regex]
+  // [START bigtable_filters_limit_col_qualifier_regex]
+  // [START bigtable_filters_limit_col_range]
+  // [START bigtable_filters_limit_value_range]
+  // [START bigtable_filters_limit_value_regex]
+  // [START bigtable_filters_limit_timestamp_range]
+  // [START bigtable_filters_limit_block_all]
+  // [START bigtable_filters_limit_pass_all]
+  // [START bigtable_filters_modify_strip_value]
+  // [START bigtable_filters_modify_apply_label]
+  // [START bigtable_filters_composing_chain]
+  // [START bigtable_filters_composing_interleave]
+  // [START bigtable_filters_composing_condition]
+  const Bigtable = require('@google-cloud/bigtable');
```

Comment:
I'm a little surprised we don't need to destructure this with `const {Bigtable} = require('@google-cloud/bigtable');`

Reply:
should I do that? didn't run into an issue with it, but let me know what would be preferred